test/unit/registration.js

```diff
@@ -103,6 +103,34 @@ describe('Returning a constructor', function () {
     expect(called).to.equal(true);
   });
 
+  it('should merge prototype functions if an element is skated more than once', function () {
+    var tag = helperElement('my-el');
+    var element = document.createElement(tag.safe);
+
+    skate(tag.safe, {
```

Comment:
Instead of having to do the run around of passing `tag.safe` to `skate()` I created another method in the `element` helper:

https://github.com/skatejs/skatejs/blob/master/test/lib/element.js#L14